theano/gpuarray/dnn.py

```diff
@@ -361,7 +361,7 @@ def do_constant_folding(self, node):
         return False
 
     def __init__(self, border_mode, subsample=(1, 1), conv_mode='conv',
-                 precision="float32"):
+                 precision="float32", precision="float64"):
```

Comment:
The Issue mentions that changing the initialization and default value worked for the user, with a patch added into the issue #5555.